### PR TITLE
[Misc] Setup Issue template form & pull request template

### PR DESCRIPTION
**Pull request by JackMoseley2001**
- Adds a form that can be used to create a new feature request in a specific format
- Adds a pull request template to allow important information to be filled out

### CODE REVIEW

**Comment by JavaRip** [comment]
Looks a bit over complicated a bits of it are a bit hard to understand

File: .github/ISSUE_TEMPLATE/new_feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: New Feature
+description: File some details about a new feature
+title: '[New Feature]: '
+labels: ['enhancement', 'help wanted']
+body:
+  - type: textarea
+    id: request-message
+    attributes:
+      label: What are you requesting?
```
Comment (JavaRip) [inline .github/ISSUE_TEMPLATE/new_feature_request.yml:9]: Requesting is also a bit non beginner friendly, should be "What affect will this have on the existing code base" or something

**Review by JavaRip — APPROVED** [review]
Look great, thanks!

**Comment by JackMoseley2001** [comment]
> Looks a bit over complicated a bits of it are a bit hard to understand

The idea behind this is to prevent just having an empty issue to start with - instead this will give a web like form to enter details into in a more structured way: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

I think from reading it's completely optional to use once implemented, but might make creating issues a fair bit quicker

**Pull request merged**